src/main.cc

```diff
@@ -51,7 +51,8 @@ void Serialization(std::ostream *stream,
     *stream << std::flush;
 }
 
-int ProcessAST(const Config& config, std::unique_ptr<std::istream>& in, std::unique_ptr<std::ostream>& out) {
```

Comment:
I thought that we use `func(prams...) {` so the opening bracket is on the same line